packages/liveblocks-core/src/comments/types/Reaction.ts

```diff
@@ -0,0 +1,9 @@
+import type { DateToString } from "./DateToString";
+
+export type Reaction = {
+  emoji: string;
+  createdAt: Date;
```

Comment:
@marcbouchenoire This type is slightly different from the `CommentReaction` type. What do you think we should name this? I feel like `Reaction` is too close to `CommentReaction` but there is a distinction between the two.

`Reaction`:
```ts
export type Reaction = {
  emoji: string;
  createdAt: Date;
  userId: string;
}
```

`CommentReaction`:
```ts
export type Reaction = {
  emoji: string;
  createdAt: Date;
  userId: CommentReactionUser[];
}

Reply:
I also feel like it should still include "comment", what do you think of something like `CommentIndividualReaction`/`CommentReactionIndividual`?

Reply:
Definitely better than just `Reaction`. We do have to be careful about the names because this type will be exported from the `node` package. @nvie Do you have any thoughts?

Reply:
My ideal choice would be something around `User` instead of `Individual`/`Single` but there's `CommentReactionUser` already (although it's not exported so we could rename this one to have a better name for the public `node` one)

Reply:
What I've done for now since our current `CommentReactionUser` is not exported, is move the type to inline:

```ts
export type CommentReaction = {
  emoji: string;
  createdAt: Date;
  users: {
    id: string;
  }[];
};
```

and renamed the new `Reaction` type to `CommentUserReaction`:

```ts
export type CommentUserReaction = {
  emoji: string;
  createdAt: Date;
  userId: string;
};
```